fuzz/server.c

```diff
@@ -507,8 +517,12 @@ int FuzzerTestOneInput(const uint8_t *buf, size_t len)
     const uint8_t *bufp;
     EVP_PKEY *pkey;
     X509 *cert;
+#ifndef OPENSSL_NO_DSA
```

Comment:
I think the NO_DSA doesn't make sense at this line (and some others), I don't think the EC(DSA) code shares anything with DSA code.